.github/workflows/prerelease.yml

```diff
@@ -10,14 +10,8 @@ on:
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   GO_VERSION: '1.9.7'
-  GPG_MAIL: 'infrastructure-eng@newrelic.com'
-  GPG_PASSPHRASE: ${{ secrets.OHAI_GPG_PASSPHRASE }}
-  GPG_PRIVATE_KEY_BASE64: ${{ secrets.OHAI_GPG_PRIVATE_KEY_BASE64 }} # base64 encoded
   INTEGRATION: "mongodb"
   ORIGINAL_REPO_NAME: 'newrelic/nri-mongodb'
```

Comment:
isn't this the same of `REPO_FULL_NAME: ${{ github.event.repository.full_name }}`

Reply:
solved offline